### ISSUE TITLE
Location of Location Tag Data?

### ISSUE THREAD

**Issue by tinglycraniumplacidly** [osmandapp/Osmand#6080]
Hello, I'm looking to manage my OsmAnd~ Favorites on my computer. I've found that it is possible to produce a gpx file that can be imported into OsmAnd~. However, the gpx file format only supports tags like cmt and desc and not ones like what the osm API calls 'opening_hours', 'phone', 'website', etc.

When you do an online search with OsmAnd~, it will give you that extra tag data like wikipedia, url, etc. I believe these are called poi values in OsmAnd~?

![osmand](https://user-images.githubusercontent.com/12286515/45922230-c7130480-be7a-11e8-810f-623c1819475a.png)

I'm wondering where this information is stored. It is present with Favorites that are saved _from_ an online search. In the above example, I can:

1.  Search for 'Union Station'
2.  Add a star to make it a Favorite
3.  Look up my Favorites and find all the additional data for that Favorite there

But there doesn't seem to be a way (yet) to associate an imported gpx location with that information. I'm wondering where this data (like 'poi_opening_hours') can be found on my phone once it's been searched and downloaded?

Basically what I'm looking for is a database that contains values listed in:

Osmand/OsmAnd/res/values/phrases.xml

Thanks!

**Comment by vshcherb** [comment]
Today it is not possible cause it is internal information linked to POI, you can produce POI map with OsmAndMapCreator if you want to have it fully customized
